docs/quickstart.md

```diff
@@ -0,0 +1,80 @@
+# Quickstart
+
+We will setup a simple django app using kubeyard built-in template.
```

Comment:
I would rename this document into quickstart_django.md or I would write down here that apart django there are other possibilities like flask.

Reply:
It's not django specific quickstart docs. Django is only app, but we want to explain how to setup `kubeyard` environment.